AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -227,6 +228,32 @@ - (void)testApplyEnabledStateWorks {
   [[MSAnalytics sharedInstance].sessionTracker stop];
 }
 
+- (void)testApplyEnabledStateWithAutoPageTrackingEnabled {
+  
+  // If
+  id analyticsMock = OCMPartialMock([MSAnalytics sharedInstance]);
+  id analyticsCategoryMock = OCMClassMock([MSAnalyticsCategory class]);
+  NSString *testPageName = @"TestPage";
+  OCMStub([analyticsCategoryMock missedPageViewName]).andReturn(testPageName);
+  [MSAnalytics setAutoPageTrackingEnabled:YES];
+  MSServiceAbstract *service = [MSAnalytics sharedInstance];
+  [MSAppCenter configureWithAppSecret:kMSTestAppSecret];
+  
+  // When
+  [[MSAnalytics sharedInstance] startWithLogManager:OCMProtocolMock(@protocol(MSLogManager))
+                                          appSecret:kMSTestAppSecret];
+
+  // FIXME: logManager holds session tracker somehow and it causes other test failures. Stop it for hack.
+  [[MSAnalytics sharedInstance].sessionTracker stop];
+
+  // Run main loop to let the block in applyEnabledState to be dispatched.
+  [[NSRunLoop mainRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:0.01]];
```

Comment:
Why not like [that](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/342c711dd182c5baac553fc722f5acabaf122ab7/AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m#L702-L709)?

Reply:
Yeah, that's better, thanks! Opened new PR #907